src/side-nav/side-nav.jsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
+import { AutoPrefix, Transitions } from '../styles';
+import Overlay from '../overlay';
+import Menu from '../menus/menu';
```

Comment:
Personal I think composable components are much better compare to embedded menu item list. 
I think this is a more react way to write code and many would prefer to do it this way. (#413,#27) It is also much neatly compare to LeftNav. 

I do realize menus are slower, but it also has much more features and easier to use. According to the online document: 
    //We're working on migrating some of our components to use a new implementation of menus.
    //If you'd like to use the new menu before our migration is complete, please directly
    //require them like this: